packages/block-library/src/navigation/block-navigation-list.js

```diff
@@ -5,38 +5,25 @@ import {
 	__experimentalBlockNavigationTree,
 	store as blockEditorStore,
 } from '@wordpress/block-editor';
-import { useSelect, useDispatch } from '@wordpress/data';
+import { useSelect } from '@wordpress/data';
 
 export default function BlockNavigationList( {
 	clientId,
 	__experimentalFeatures,
 } ) {
-	const { blocks, selectedBlockClientId } = useSelect(
-		( select ) => {
-			const {
-				getSelectedBlockClientId,
-				__unstableGetClientIdsTree,
-			} = select( blockEditorStore );
-
-			return {
-				blocks: __unstableGetClientIdsTree( clientId ),
-				selectedBlockClientId: getSelectedBlockClientId(),
-			};
-		},
+	const blocks = useSelect(
+		( select ) =>
+			select( blockEditorStore ).__unstableGetClientIdsTree( clientId ),
 		[ clientId ]
 	);
 
-	const { selectBlock } = useDispatch( blockEditorStore );
-
 	return (
 		<__experimentalBlockNavigationTree
```

Comment:
Just noticing an inconsistency in naming when we import this.  In edit-site we drop the experimental prefix and here we do not.  I wonder if the naming convention of experimental imports should be more consistent across the repo?

Reply:
I always drop the __experimental and __unstable when importing. I'm not sure if that's the preferred way in Gutenberg though.